src/Compilers/Core/Portable/Operations/Operation.cs

```diff
@@ -145,9 +136,7 @@ protected void SetParentOperation(IOperation? parent)
                 ((Operation)parent).OwningSemanticModel == null || OwningSemanticModel == null);
```

Comment:
>// explicit cast is not allowed, so using "as" instead [](start = 12, length = 54)

Is this comment no longer relevant?

Reply:
(Note to future self, codeflow is showing this wrong, it should be on the comment removal in SetParentOperation below)

That comment felt pretty self-explanatory to me, explaining the what without the why.

---
In reply to: [527721421](https://github.com/dotnet/roslyn/pull/49453#discussion_r527721421) [](ancestors = 527721421)